app_builder/lib/app_builder/macos.ex

```diff
@@ -109,7 +109,7 @@ defmodule AppBuilder.MacOS do
 
     File.mkdir_p!("tmp")
     launcher_src_path = "tmp/Launcher.swift"
-    File.write!(launcher_src_path, launcher())
+    File.write!(launcher_src_path, launcher(["/rel/vendor/bin"]))
```

Comment:
```suggestion
    additional_paths = Keywword.get(options, :additional_paths, [])
    File.write!(launcher_src_path, launcher(additional_paths))
```

Don't forget to add `:addtional_paths` to `Keyword.validate!` too.

And then change the `mix.exs` to pass something like:

    additional_paths: ["rel/vendor/elixir/bin"]

You could even pass the erts too, if you remove the version:

    additional_paths: ["rel/vendor/erts/bin", "rel/vendor/elixir/bin"]

And this way we no longer need to rely on `rel/vendor/bin`, we pass any additional path we might want!